src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -890,6 +946,74 @@ MulticopterPositionControl::failsafe(vehicle_local_position_setpoint_s &setpoint
 	}
 }
 
+void
+MulticopterPositionControl::update_avoidance_waypoint_desired(PositionControlStates &states)
+{
+	_traj_wp_avoidance_desired = _flight_tasks.getAvoidanceWaypoint();
+
+	_traj_wp_avoidance_desired.timestamp = hrt_absolute_time();
+	_traj_wp_avoidance_desired.type = vehicle_trajectory_waypoint_s::MAV_TRAJECTORY_REPRESENTATION_WAYPOINTS;
+
+	states.position.copyTo(_traj_wp_avoidance_desired.waypoints[vehicle_trajectory_waypoint_s::POINT_0].position);
```

Comment:
is point 0 of the information to the avoidance defined as the current vehicle state?
if yes it's correct.

Reply:
correct